core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -486,31 +490,49 @@ var ExplorationEditorTranslationTab = function() {
   };
 
   this.expectContentAccessibilityToMatch = async function(content) {
+    await waitFor.visibilityOf(
+      translationTabContentAccessibility,
+      'Translation tab content taking too long to appear');
     expect(await translationTabContentAccessibility.getAttribute(
       'aria-label')).toMatch(content);
   };
 
   this.expectFeedbackAccessibilityToMatch = async function(content) {
+    await waitFor.visibilityOf(
+      translationTabFeedbackAccessibility,
+      'Translation tab feedback taking too long to appear');
     expect(await translationTabFeedbackAccessibility.getAttribute(
       'aria-label')).toMatch(content);
   };
 
   this.expectHintAccessibilityToMatch = async function(content) {
+    await waitFor.visibilityOf(
+      translationTabHintAccessibility,
+      'Translation tab hint takint too long to appear');
     expect(await translationTabHintAccessibility.getAttribute(
       'aria-label')).toMatch(content);
   };
 
   this.expectSolutionAccessibilityToMatch = async function(content) {
+    await waitFor.visibilityOf(
+      translationTabSolutionAccessibility,
+      'Translation tab solution taking too long to appear');
     expect(await translationTabSolutionAccessibility.getAttribute(
       'aria-label')).toMatch(content);
   };
 
   this.expectStartRecordingAccessibilityToMatch = async function(content) {
+    await waitFor.visibilityOf(
+      translationTabStartRecordingAccessibility,
+      'Translation tab start recording taking too long to appear');
     expect(await translationTabStartRecordingAccessibility.getAttribute(
       'aria-label')).toMatch(content);
   };
 
   this.expectUploadRecordingAccessibilityToMatch = async function(content) {
+    await waitFor.visibilityOf(
+      translationTabUploadRecordingAccessibility,
+      'Translation tab upload recording taking too long to appear');
     expect(await translationTabUploadRecordingAccessibility.getAttribute(
       'aria-label')).toMatch(content);
```

Comment:
Can we define action.getAttribute and use it here?

Reply:
I have added this function in action.js
![Screenshot from 2021-09-13 14-04-35](https://user-images.githubusercontent.com/42772952/133051292-2f1564a8-1b01-42cc-b500-d0973c654ffa.png)

It is working for other elements but not working for below element
![Screenshot from 2021-09-13 14-03-21](https://user-images.githubusercontent.com/42772952/133051166-a3730177-0413-4257-a5e4-43ff90be63d3.png)